atlasdb-config/src/main/java/com/palantir/atlasdb/http/AtlasDbHttpClients.java

```diff
@@ -64,6 +64,13 @@
             new ConnectionSpec.Builder(ConnectionSpec.MODERN_TLS)
                     .tlsVersions(TlsVersion.TLS_1_2)
                     .cipherSuites(
+                            // This GCM cipher suite is for HTTP/2 over TLS1.2 as clients have to
+                            // have at least one permitted cipher suite not in the blacklist. The
+                            // timelock server will support only HTTP/2 connections, and this will ensure
```

Comment:
We're not actually preventing the fallback right? So a client that doesn't support HTTP/2 is actually still being supported.